app/(afterLogin)/(onboarding)/onboarding/_components/steper/timezone.tsx

```
@@ -2,7 +2,9 @@
 import React, { useEffect, useState } from 'react';
 // import TimezoneComponent from '@/app/(afterLogin)/(timesheetInformation)/timesheet/settings/time-zone/_components/timeZone';
 import useStepStore from '@/store/uistate/features/organizationStructure/steper/useStore';
-import { Button } from 'antd';
+import { Button, Typography } from 'antd';
+
+const {} = Typography;
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)